src/oge/output_data.py

```diff
@@ -93,7 +93,7 @@ def zip_results_for_s3(year):
     # move and rename the plant attributes files
     shutil.copy(
         f"{results_folder()}/{year}/plant_data/plant_static_attributes.csv",
-        f"{data_folder()}/s3_upload/plant_static_attributes_{year}.csv",
+        f"{data_folder()}/s3_upload/{year}_plant_static_attributes.csv",
```

Comment:
We need to keep the naming convention with the year at the end of the name - that is what the website is looking for.

Reply:
Good catch. It was part of the original commit and the rebase left it untouched

Reply:
Changed